libffi-dev is required to make on ubuntu

With the introduction of https://github.com/gratipay/gratipay.com/commit/01b4ec3965999be45cd88229c0faa30ab5cc2adc I'm currently unable to `make` on my local gratipay.com because of the following error:

```
Installing collected packages: cryptography, ipaddress, idna, enum34, cffi, pycparser, boto3, botocore, jmespath, python-dateutil, docutils, futures, braintree, misaka, Babel, pytz, invoke, environment, honcho, libsass, six, fake-factory, raven, balanced, uritemplate, iso8601, wac, xmltodict, requests-oauthlib, requests, oauthlib, chardet, pyasn1, certifi, simplejson, postgres, psycopg2, aspen-jinja2, Jinja2, MarkupSafe, gunicorn, aspen, filesystem-tree, algorithm, dependency-injection, first, python-mimeparse
  Running setup.py install for cryptography
    c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory
     #include <ffi.h>
                     ^
```

Looks like simply `sudo apt-get install libffi-dev libssl-dev` fixed it.

--- a/README.md
+++ b/README.md
@@ -106,7 +106,7 @@ met](http://initd.org/psycopg/docs/faq.html#problems-compiling-and-deploying-psy
 On Debian or Ubuntu you will need the following packages:
 
 ```shell
-sudo apt-get install postgresql-9.3 postgresql-contrib libpq-dev python-dev
+sudo apt-get install postgresql-9.3 postgresql-contrib libpq-dev python-dev libffi-dev libssl-dev
 ```
 
 On OS X you can [download Postgres directly](http://www.postgresql.org/download/macosx/) or install through [Homebrew](http://brew.sh/):

--- a/scripts/bootstrap-debian.sh
+++ b/scripts/bootstrap-debian.sh
@@ -10,7 +10,9 @@ echo "[*] Installing base dependencies..
       postgresql
       postgresql-contrib  - pg_trgm, pg_stat_statements
       python
-      python-dev  - for building misaka"
+      python-dev  - for building misaka
+      libffi-dev 
+      libssl-dev  - for building cryptography"
 sudo apt-get update
 sudo apt-get install -y \
     g++ \
@@ -21,7 +23,9 @@ sudo apt-get install -y \
     postgresql-contrib \
     python \
     language-pack-en \
-    python-dev
+    python-dev \
+    libffi-dev \
+    libssl-dev
 
 echo "[*] Installing dependencies for tests..
       default-jre     - jstests need java to run selenium

